qfieldsync/utils/file_utils.py

```diff
@@ -79,3 +80,11 @@ def open_folder(path):
         subprocess.Popen(["open", '-R', path])
     else:
         subprocess.Popen(["xdg-open", path])
+
+
+def import_file_checksum(folder):
+    path = os.path.join(folder, "data.gpkg")
+    if not os.path.exists(path):
+        path = os.path.join(folder, "data.sqlite")
+    file_data = open(path, 'rb').read()
```

Comment:
This will produce a `ResourceWarning` no?

Better safe than sorry:

```suggestion
    with open(path, 'rb') as f:
        file_data = f.read()
```